flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
It doesn't work on other architectures?

Reply:
A pre-compiled version of ``Qt for Python`` 5.9 for x86_64 is used. I tried to compile version 5.11 from source with the KDE runtime, but wasn't able to get it working.

Reply:
I think that the problem is caused by the older versions of Clang/LLVM in the SDK.

Reply:
That's annoying, but understandable. This isn't a blocker - but could you try with the upcoming 1.8 FDo SDK? https://gitlab.com/freedesktop-sdk/freedesktop-sdk it's currently unstable but real world feedback like this would be great.